addons/mail/static/src/new/core/thread_model.js

```diff
@@ -124,6 +123,18 @@ export class Thread {
         this.messages.sort((msgId1, msgId2) => msgId1 - msgId2);
     }
 
+    get channelMembers() {
+        const members = [];
+        const allChannelMembers = this._state.channelMembers;
+        for (const channelMemberId in allChannelMembers) {
+            const member = allChannelMembers[channelMemberId];
+            if (member.threadId === this.id) {
+                members.push(member);
+            }
+        }
+        return members;
+    }
```

Comment:
This looks equivalent to simply
```suggestion
    get channelMembers() {
        return this._state.channelMembers.filter((m) => m.threadId === this.id);
    }
```

Reply:
Note that I'm not sure if using a getter here is a good idea, as there might be a lot of members in a lot of channel, this might quickly lead to slowness.

I would suggest to manually synchronize members of the given channel when inserting (or deleting) a member instead.